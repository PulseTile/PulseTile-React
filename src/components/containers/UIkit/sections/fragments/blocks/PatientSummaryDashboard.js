import React from 'react';

/**
 * This component returns content of PatientSummaryDashboard section in Blocks
 */
const PatientSummaryDashboard = () => {
  return (
    <div id="dashboard" className="ui-section">
      <strong className="ui-title">Patient Summary Dashboard</strong>
      <div className="ui-sub-section">
        <div className="dashboard">
          <div className="dashboard-item">
            <div className="board">
              <div className="board-header">
                <div className="control-group right">
                  <button className="btn btn-success btn-inverse btn-board-more">
                    <i className="btn-icon fa fa-caret-right"></i>
                  </button>
                </div>
                <h3 className="board-title">Dashboard title</h3>
              </div>
              <div className="board-body">
                <ul className="board-list">
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div className="dashboard-item">
            <div className="board">
              <div className="board-header">
                <div className="control-group right">
                  <button className="btn btn-success btn-inverse btn-board-more">
                    <i className="btn-icon fa fa-caret-right"></i>
                  </button>
                </div>
                <h3 className="board-title">Dashboard title</h3>
              </div>
              <div className="board-body">
                <ul className="board-list">
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link"></span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div className="dashboard-item">
            <div className="board">
              <div className="board-header">
                <div className="control-group right">
                  <button className="btn btn-success btn-inverse btn-board-more">
                    <i className="btn-icon fa fa-caret-right"></i>
                  </button>
                </div>
                <h3 className="board-title">Dashboard title</h3>
              </div>
              <div className="board-body">
                <ul className="board-list">
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link"></span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link"></span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div className="dashboard-item">
            <div className="board">
              <div className="board-header">
                <div className="control-group right">
                  <button className="btn btn-success btn-inverse btn-board-more">
                    <i className="btn-icon fa fa-caret-right"></i>
                  </button>
                </div>
                <h3 className="board-title">Dashboard title</h3>
              </div>
              <div className="board-body">
                <ul className="board-list">
                  <li className="board-list-item">
                    <span className="board-list-link">Dashboard item</span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link"></span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link"></span>
                  </li>
                  <li className="board-list-item">
                    <span className="board-list-link"></span>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default PatientSummaryDashboard;
