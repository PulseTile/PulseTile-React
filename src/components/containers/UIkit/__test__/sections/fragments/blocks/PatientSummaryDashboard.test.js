import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import PatientSummaryDashboard from '../../../../sections/fragments/blocks/PatientSummaryDashboard';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <PatientSummaryDashboard />', () => {
  it('should renders PatientSummaryDashboard with props correctly', () => {
    const component = shallow(<PatientSummaryDashboard />);
    expect(component).toMatchSnapshot();
  });
});
