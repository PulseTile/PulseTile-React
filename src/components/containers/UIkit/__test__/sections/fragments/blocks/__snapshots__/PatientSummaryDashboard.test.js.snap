// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <PatientSummaryDashboard /> should renders PatientSummaryDashboard with props correctly 1`] = `
<div
  className="ui-section"
  id="dashboard"
>
  <strong
    className="ui-title"
  >
    Patient Summary Dashboard
  </strong>
  <div
    className="ui-sub-section"
  >
    <div
      className="dashboard"
    >
      <div
        className="dashboard-item"
      >
        <div
          className="board"
        >
          <div
            className="board-header"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-board-more"
              >
                <i
                  className="btn-icon fa fa-caret-right"
                />
              </button>
            </div>
            <h3
              className="board-title"
            >
              Dashboard title
            </h3>
          </div>
          <div
            className="board-body"
          >
            <ul
              className="board-list"
            >
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div
        className="dashboard-item"
      >
        <div
          className="board"
        >
          <div
            className="board-header"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-board-more"
              >
                <i
                  className="btn-icon fa fa-caret-right"
                />
              </button>
            </div>
            <h3
              className="board-title"
            >
              Dashboard title
            </h3>
          </div>
          <div
            className="board-body"
          >
            <ul
              className="board-list"
            >
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                />
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div
        className="dashboard-item"
      >
        <div
          className="board"
        >
          <div
            className="board-header"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-board-more"
              >
                <i
                  className="btn-icon fa fa-caret-right"
                />
              </button>
            </div>
            <h3
              className="board-title"
            >
              Dashboard title
            </h3>
          </div>
          <div
            className="board-body"
          >
            <ul
              className="board-list"
            >
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                />
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                />
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div
        className="dashboard-item"
      >
        <div
          className="board"
        >
          <div
            className="board-header"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-board-more"
              >
                <i
                  className="btn-icon fa fa-caret-right"
                />
              </button>
            </div>
            <h3
              className="board-title"
            >
              Dashboard title
            </h3>
          </div>
          <div
            className="board-body"
          >
            <ul
              className="board-list"
            >
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                >
                  Dashboard item
                </span>
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                />
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                />
              </li>
              <li
                className="board-list-item"
              >
                <span
                  className="board-list-link"
                />
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
